### PR TITLE
Mishal create card pr

### PR DESCRIPTION
**Pull request by misusonu18**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1,4 +1,31 @@
-var navbarHtml = `
+window.onload = function () {
+    checkAuth();
+}
+
+const checkAuth = () => {
+	let userEmail = user = '';
```
Comment (hirenkeraliya) [inline js/app.js]: tab?

File: js/manage.js [hunk 2]
```diff
@@ -1,125 +1,175 @@
...
+const submitForm = async () => {
+    checkValidation();
+
+    if (phoneNumberFlag == true && emailFlag == true && nameFlag == true && websiteFlag == true && githubFlag == true) {
+        submitButton.setAttribute('disabled', true);
+		displayLoading();
+        await axios.post(
+            '/.netlify/functions/fetchPhoneNumbers',
+		).then((res) => {
+			fileName = (nameInput.value).replace(' ', '_') + '_' + seconds + '.json';
```
Comment (hirenkeraliya) [inline js/manage.js]: tab?

File: js/manage.js [hunk 3]
```diff
@@ -1,125 +1,175 @@
...
+            'sha': shaName,
+            'data': phoneNumberArray
+        }).then(() => {
+			createData();
+		}).catch(() => {
+			hideLoading();
+			submitButton.removeAttribute('disabled');
+		})
+	;
+}
```
Comment (hirenkeraliya) [inline js/manage.js]: proper indentation by spaces.

File: js/manage.js [hunk 4]
```diff
@@ -1,125 +1,175 @@
...
-let submitForm = async () => {
-	if (phoneNumberFlag == true && emailFlag == true && firstNameFlag == true && lastNameFlag == true && websiteFlag == true && githubFlag == true) {
-		//
-	}
+const updateThePhoneNumber = (shaName, phoneNumberArray) => {
+    phoneNumberArray.push(
+        { 'file_name': fileName, 'folder_name': userName, 'phone_number': parseInt(phoneNumberInput.value) }
+    );
+    axios.post(
+        '/.netlify/functions/updatePhoneNumbersList', {
```
Comment (hirenkeraliya) [inline js/manage.js]: use netlify redirects.

**Review by hirenkeraliya — CHANGES_REQUESTED** [review]

File: Readme.md [hunk 5]
```diff
@@ -1,3 +1,10 @@
...
-[![Netlify Status](https://api.netlify.com/api/v1/badges/af2c10bc-a7d4-4a4f-913a-c62f02f002e5/deploy-status)](https://app.netlify.com/sites/digital-visiting-demo/deploys)
\ No newline at end of file
+[![Netlify Status](https://api.netlify.com/api/v1/badges/af2c10bc-a7d4-4a4f-913a-c62f02f002e5/deploy-status)](https://app.netlify.com/sites/digital-visiting-demo/deploys)
+
+## Setup
+1. Clone The Repo: `git@github.com:misusonu18/digital-visiting-card-js.git`
+
+2. Create .env file from the example file: `php -r "file_exists('.env') || copy('.env.example', '.env');"`
+
+3. Setup the variable
```
Comment (hirenkeraliya) [inline Readme.md]: Add steps for the deployement

File: js/manage.js [hunk 6]
```diff
@@ -1,125 +1,175 @@
...
+
+        phoneNumberValidation.innerText = '';
+        phoneNumberFlag = true;
+    }
+};
+
+const submitForm = async () => {
+    checkValidation();
+
+    if (phoneNumberFlag == true && emailFlag == true && nameFlag == true && websiteFlag == true && githubFlag == true) {
```
Comment (hirenkeraliya) [inline js/manage.js]:
```
if (phoneNumberFlag != true || emailFlag != true || nameFlag != true || websiteFlag != true || githubFlag != true) {
    return;
}
```

**Review by hirenkeraliya — CHANGES_REQUESTED** [review]

**Review by hirenkeraliya — APPROVED** [review]

File: functions/updatePhoneNumbersList.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+                responseMessage = response;
+            })
+            .catch(function (error) {
+                responseMessage = error;
+            });
+
+        return {
+            statusCode: 201,
+            body: JSON.stringify({
+                msg: (responseMessage.data.content.name == 'validate_phone_numbers.json') ? "Successfull File Updated" : "Something Went Wrong",
```
Comment (gauravmak) [inline functions/updatePhoneNumbersList.js]: Check message

**Review by gauravmak — CHANGES_REQUESTED** [review]

**Pull request merged**